fibsem/structures.py

```diff
@@ -1097,6 +1097,7 @@ class FibsemBitmapSettings(FibsemPatternSettings):
     path: InitVar[Optional[Union[str, os.PathLike]]] = None
     array: InitVar[Optional[NDArray[Any]]] = None
     bitmap: Optional[NDArray[Any]] = field(init=False)
+    interpolate: Optional[Literal["nearest", "bicubic", "bilinear"]] = None
```

Comment:
Ideally, this would be an Enum but guessing it's to avoid the gui complexity that comes with that right?

Reply:
It's mostly to avoid adding an enum that's used in one place for one argument. Since FibsemBitmapSettings is only used internally, unlike `BitmapPattern`, it's easy enough to ensure the correct string is passed (using Literal means type checking will complain if not).

It would be good to simplify the `interpolate` and `bicubic` arguments in `BitmapPattern` and `TrenchBitmapPattern` to create a drop down list based on the typing (from either a Literal or Enum) at some point in the future though, but I didn't think it was worth creating another object until the GUI stuff was figured out.